dokku-vm/azuredeploy.json

```diff
@@ -69,17 +100,16 @@
         },
         "allowedValues": [
             "East US",
-            "East US 2",
-            "East Asia",
-            "West US",
```

Comment:
Weird. Anyway, running the official CLI client—with `azure location list --json`—gave me a bigger list which I committed up.